packages/legacy-client/src/client/index.ts

```diff
@@ -14,18 +14,427 @@
  * limitations under the License.
  */
 
-export * from "./actions/index.js";
-export * from "./baseTypes/index.js";
-export * from "./errors/index.js";
-export * from "./foundryClient.js";
-export * from "./foundryClientOptions.js";
-export * from "./interfaces/index.js";
-export * from "./objects.js";
-export * from "./objectSets/index.js";
-export * from "./ontology.js";
-export * from "./OntologyMetadata.js";
-export * from "./OsdkLegacyObject.js";
-export * from "./Page.js";
+export type {
```

Comment:
Some of these files exported an identifier whose name was reused in a different file. This meant that types could not properly be resolved. I manually listed everything that needed to be listed to get around this type  of error.